### PR TITLE
Fix the wrong CI commands I introduced earlier

### PR DESCRIPTION
**Pull request by poliorcetics**

### CODE REVIEW

**Comment by arijit79** [comment]
You have misspelt cargo with cg

**Comment by poliorcetics** [comment]
`git` found a merge conflict for some reasons, and I copy-pasted from my test terminal so I was using my local aliases. 🤦

**Comment by poliorcetics** [comment]
There is no output terminal available for the example, let's only check it.

**Comment by arijit79** [comment]
Actually, you should also just check static output, as the quitting depends on the rows of the terminal and we don't know it in the CI

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -28,7 +28,27 @@ jobs:
...
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+    # Those two needs human intervention to stop so only check them.
+    - name: Tokio
+      run: cargo check --example=dyn_tokio --features=tokio_lib
+    - name: Async-std
+      run: cargo check --example=dyn_async_std --features=async_std_lib
+    # This one can run without human intervention.
```
Comment (poliorcetics) [inline .github/workflows/ci.yml:52]:
```suggestion
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -28,7 +28,27 @@ jobs:
...
+    - name: Static
+      run: cargo run --example=static --features=static_output
```
Comment (poliorcetics) [inline .github/workflows/ci.yml:54]:
```suggestion
      run: cargo check --example=static --features=static_output
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -28,7 +28,27 @@ jobs:
...
+  examples:
+    name: examples
+    env:
+      # Emit backtraces on panics.
+      RUST_BACKTRACE: 1
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+    # Those two needs human intervention to stop so only check them.
```
Comment (poliorcetics) [inline .github/workflows/ci.yml:47]:
```suggestion
    # No terminal available in CI, only check the examples.
```

**Comment by poliorcetics** [comment]
Nice it passed ! I'll need this to be merged before opening a PR for the tests so that CI don't say it's broken when it's not.

**Comment by arijit79** [comment]
Thanks for the PR. One last thing before merging, could you squash all these commits

**Comment by arijit79** [comment]
Merging the PR. I am really thankful to you

**Pull request merged**